### ISSUE TITLE
[A11Y] Search select fields missing form label

### ISSUE THREAD

**Issue by israelcefrin** [pkp/immersion#133]
## Issue Description
Form control element <select> has no associated label, it will not be announced correctly to users that rely on assistive technology as screenreaders.

## How to reproduce / Current behavior
1. Access the search page of a Journal that uses Immersion Theme. https://ojs34.demo.publicknowledgeproject.org/index.php/immersion/search
2. Activate a screenreader and navigate to the select field
3. Notice that the field name are not announced.

## What application are you using?
 OJS version 3.4

## Stack
- Device: Laptop
- OS:  MacOS Sequoia 
- Browser: Firefox

## Testing method
- Method:  Manual, Keyboard 
- Screenreader:   VoiceOver

## Action
Add the attribute `id` to the select fields to make them relatable to the declared `labels`.
